src/devices/bus/nes/jaleco.cpp

```diff
@@ -315,11 +315,11 @@ void nes_jf17_device::write_h(offs_t offset, u8 data)
 
  -------------------------------------------------*/
 
-void nes_ss88006_device::device_timer(emu_timer &timer, device_timer_id id, int param)
+TIMER_CALLBACK_MEMBER(nes_ss88006_device::irq_timer_tick)
 {
-	if (id == TIMER_IRQ)
+	if (m_irq_enable)
 	{
-		if (m_irq_enable)
+		if (m_irq_mode & 0x08)  // 4bits counter
```

Comment:
@MooglyGuy Can you review this callback's change? It looks like it got messed up when merging my recent changes. Outer `if (m_irq_enable)` is fine, but this inner `if` shouldn't be there. Instead, body of `if (m_irq_enable)` should just have indentations corrected, it seems.

Reply:
Thanks, I'll have a look. A couple of the merges have been a bit squirrelly, but this seems to be one that I missed.

Reply:
@0kmg The mangled merged has been fixed, thanks again for spotting this!

Reply:
NP. On the contrary, thank you for the code clean up!